tests/dragonfly/acl_family_test.py

```diff
@@ -11,4 +12,40 @@ async def test_acl_list_default_user(async_client):
     """
     result = await async_client.execute_command("ACL LIST")
     assert 1 == len(result)
-    assert "user default on nopass +@all" == result[0]
+
+
+def assert_any_of(assertion, result):
+    assert assertion == result[0] or assertion == result[1]
```

Comment:
small nit: better have the function return bool and assert its condition so you can see the failed line

Reply:
This is not a `small nit` but an essential. I was very dissatisfied that it would not print the line and I wanted to comment on that. Thankfully you did first + I think your other comment using `in` is far more better so I opted in for that